elements/map/helpers.ts

```diff
@@ -237,10 +238,15 @@ export async function registerProjectionFromCode(code: string | number) {
  */
 export function registerProjection(
   name: string,
-  projection: string | proj4.ProjectionDefinition
+  projection: string | proj4.ProjectionDefinition,
+  extent?: number[]
 ) {
   proj4.defs(name, projection);
   register(proj4);
+  if (typeof extent !== "undefined") {
+    const projection = getProj(name);
```

Comment:
`projection` is already defined as function param, could we use another variable name here?

Reply:
Or maybe it is enough to just do `getProj(name).setExtent(extent)`?

Reply:
good points, i updated it accordingly